test/unit/lifecycle/properties.js

```diff
@@ -219,35 +219,98 @@ describe('lifecycle/properties', function () {
     expect(el.getAttribute('prop1')).to.equal('');
   });
 
-  describe('270', function () {
-    function setup (initialValue) {
-      var el = elem.create();
-      el.textContent = 'existing content';
+  it('should not be triggered on initialisation', function () {
```

Comment:
@nicolasv, @cyberdash this refers to where we were saying that the `update()` callback should not be called for the property initialisation.